src/components/PharmacyCard/PharmacyCard.tsx

```diff
@@ -17,15 +20,26 @@ interface PharmacyProps {
 }
 
 export function PharmacyCard(props: PharmacyProps) {
+  const { t, i18n } = useTranslation();
   const [shouldShowMap, setShouldShowMap] = useState(false);
+  function FormatDate(date: Date) {
+    // alert(i18n.language);
+    if (i18n.language === "fr") {
+      return format(date, "d MMM y", { locale: frCA });
+    }
+    return format(date, "MMM d, y", { locale: enCA });
+  }
   const availabilityMarkup = () => {
     if (props.booking) {
       return (
         <Banner status="success">
           <p>
-            <strong>Appointments available</strong>
+            <strong>{t("appointmentsavailable")}</strong>
             {props.lastUpdated.length > 0 && (
-              <> as of {format(new Date(props.lastUpdated), "MMM d, y")}</>
+              <>
+                {" "}
+                {t("asof")} {FormatDate(new Date(props.lastUpdated))}
```

Comment:
We should use i18next's interpolation to include the formatted date as part of the translation string itself (https://www.i18next.com/translation-function/interpolation).
In general, we don't know if other language translations might have the date midway through the statement, so we shouldn't rely on tacking it on the end.

Reply:
Switched to using i18next interpolation, note I need to configure the date-fns-tz format function in the i18next configuration because i18next doesn't provide a date formatting library (it suggests moment but moment is outdated)